docs/commands/ts.range.md

```diff
@@ -7,7 +7,7 @@ TS.RANGE key fromTimestamp toTimestamp
          [FILTER_BY_TS ts...]
          [FILTER_BY_VALUE min max]
          [COUNT count] 
-         [[ALIGN value] AGGREGATION aggregator bucketDuration [EMPTY]]
+         [[ALIGN value] AGGREGATION aggregator bucketDuration [BUCKETTIMESTAMP bt] [EMPTY]]
```

Comment:
@OfirMos can we include the relevant description of `BUCKETTIMESTAMP` in the command description.